terraform/shared/database/main.tf

```diff
@@ -0,0 +1,23 @@
+###
+# Target space/org
+###
+
+data "cloudfoundry_space" "space" {
+  org_name = var.cf_org_name
+  name     = var.cf_space_name
+}
+
+###
+# RDS instance
+###
+
+data "cloudfoundry_service" "rds" {
+  name = "aws-rds"
+}
+
+resource "cloudfoundry_service_instance" "rds" {
+  name             = "ohs-ct-rds-${var.env}"
+  space            = data.cloudfoundry_space.space.id
+  service_plan     = data.cloudfoundry_service.rds.service_plans["micro-psql"]
```

Comment:
[thought] This is good for now, but we should make a note to potentially pass a variable for the plan type once we add a production environment.